rpc/trace_test.go

```diff
@@ -77,7 +78,14 @@ func TestTransactionTrace(t *testing.T) {
 		if err != nil {
 			require.Equal(t, test.ExpectedError, err)
 		} else {
-			require.Equal(t, test.ExpectedResp, resp)
+			invokeTrace := resp.(InvokeTxnTrace)
+			require.Equal(t, invokeTrace.ExecuteInvocation, test.ExpectedResp.ExecuteInvocation)
+			require.Equal(t, invokeTrace.Type, test.ExpectedResp.Type)
+			require.Equal(t, invokeTrace.StateDiff, test.ExpectedResp.StateDiff)
+			require.Equal(t, invokeTrace.FeeTransferInvocation, test.ExpectedResp.FeeTransferInvocation)
+			require.Equal(t, invokeTrace.ExecutionResources, test.ExpectedResp.ExecutionResources)
+			require.NotNil(t, invokeTrace.ExecutionResources)
+			fmt.Println(invokeTrace.ExecutionResources)
```

Comment:
If the fields and types of `invokeTrace` and `test`, I think `require.Equal` is sufficient.